client/components/date-range/header.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+import { noop } from 'lodash';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import SharedProps from './shared-props';
+
+interface Props {
+	onApplyClick: () => void;
+	onCancelClick: () => void;
+	applyButtonText: string | null | undefined;
+	cancelButtonText: string | null | undefined;
+}
+
+const DateRangeHeader: FunctionComponent< Props & SharedProps & LocalizeProps > = props => {
```

Comment:
Without `localize` HOC, we no longer expect `LocalizeProps`:

```suggestion
const DateRangeHeader: FunctionComponent< Props & SharedProps > = props => {
```

Reply:
Good catch! Sorry, I must have lost this when I redid my changes because of a messed-up rebase.